test/controllers/users_controller_test.rb

```diff
@@ -115,8 +115,13 @@ class UsersControllerTest < ActionController::TestCase
 
   test 'should update profile text' do
     sign_in users(:standard_user)
-    patch :update_profile, params: { user: { profile_markdown: 'ABCDEF GHIJKL', website: 'https://example.com/user',
-                                             twitter: '@standard_user' } }
+    patch :update_profile, params: {
+      user: {
+        profile_markdown: 'ABCDEF GHIJKL',
+        website: 'https://example.com/user',
+        twitter: 'standard_user'
+      }
+    }
```

Comment:
Is this correct now that we've changed to generic links, or does this need updating to use the new format?

Reply:
No, it's definitely not correct anymore, I only fixed it up temporarily by dropping the Twitter handle validation check, the test needs to be updated (cc @cellio)

Reply:
I can remove website and twitter and add discord (which is missing).  I might need a little consultation on how to test for the new user_websites table.

Reply:
Added test for the new values.